typescript/web/src/utils/testing/mock-data.ts

```diff
@@ -0,0 +1,10 @@
+export const mockWorkspaceSimple = {
```

Comment:
* Can we call the file `data.fixtures.ts`?
* Can you use CONSTANT_CASE for deep-readonly values?
* Why didn't you reused the GraphQL types defined in `@labelflow/graphql-types`?

Reply:
> * Can you use CONSTANT_CASE for deep-readonly values?
What do you mean?